bin/docker-compose

```diff
@@ -49,41 +38,35 @@ function __main__() {
   if [[ "$SIBLING_CONTAINERS_ENABLED" == "true" ]]; then
     compose_file_flags+=("-f $TOOLKIT_ROOT/lib/docker-compose.sibling-containers.yml")
   fi
-  if [[ "${NGINX_ENABLED:-false}" == "true" ]]; then
+  if [[ "${NGINX_ENABLED}" == "true" ]]; then
     compose_file_flags+=("-f $TOOLKIT_ROOT/lib/docker-compose.nginx.yml")
   fi
-
+  if [[ "${GIT_BRIDGE_ENABLED}" == "true" ]]; then
```

Comment:
Should we also check whether the version actually supports git-bridge feature?

Reply:
Added check in https://github.com/overleaf/toolkit/pull/146/commits/48349f3607271ad0020000994e3ea47cac7f1b02

I also check that Server Pro is enabled.